client/spec/authoring_spec.js

```diff
@@ -4,7 +4,7 @@
 var workspace = require('./helpers/workspace');
 
 describe('authoring', function() {
-    it('can open item stage', function() {
+    xit('can open item stage', function() { // obsolete?
```

Comment:
This should be resolved before merging IMO. Either we keep the test or remove it completely.